cime_config/machines/config_compilers.xml

```diff
@@ -321,7 +321,7 @@ flags should be captured within MPAS CMake files.
   <FFLAGS>
     <base>  -convert big_endian -assume byterecl -ftz -traceback -assume realloc_lhs -fp-model source </base>
     <append compile_threaded="TRUE"> -qopenmp </append>
-    <append DEBUG="TRUE"> -O0 -g -check uninit -check bounds -check pointers -fpe0 -check noarg_temp_created </append>
+    <append DEBUG="TRUE"> -O0 -g -check uninit -check bounds -check pointers -check noarg_temp_created </append>
```

Comment:
Same here. Is it the problem you mentioned with land? Could you then please explain in more detail, I am rather sure such PR should not touch flags (esp. DEBUG flags below). Thanks!